job.js

```diff
@@ -30,6 +49,10 @@ class BigQueryJob {
         this.costThresholdInGB = opts.costThresholdInGB != null ? opts.costThresholdInGB : COST_THRESHOLD_IN_GB;
         this.costPerTB = opts.costPerTB != null ? opts.costPerTB : COST_PER_TB;
 
+        this.destinationTableConfig = opts.destinationTableConfig;
+        this.writeDisposition = opts.writeDisposition;
```

Comment:
I would include the `writeDisposition` inside the `destinationTableConfig`.  It shouldn't be used otherwise